src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeFinaliserServiceTest.java

```diff
@@ -63,12 +63,12 @@ public void tearDown() {
     }
 
     @Test
-    public void finaliseEnvelope_should_update_envelope_status_and_clear_all_ocr_data() {
+    public void finaliseEnvelope_should_update_envelope_status_and_clear_all_ocr_data_and_warnings() {
         // given
         Envelope envelope = envelope(
             "JURISDICTION1",
             Status.NOTIFICATION_SENT,
-            createScannableItemsWithOcrData(3)
+            createScannableItemsWithOcrDataAndWarnings(3)
```

Comment:
I think you will need to have two test scenarios
One with ocr and warnings
One with only ocr

You won’t always have ocr with warnings.

Reply:
Have updated